src/transactions/feed/TransferFeedIcon.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import ContactCircle from 'src/components/ContactCircle'
+import CoinbasePayIcon from 'src/icons/CoinbasePayIcon'
+import { coinbasePaySendersSelector } from 'src/recipients/reducer'
+import { FeedTokenTransfer } from 'src/transactions/feed/TransferFeedItem'
+import { useTransactionRecipient } from 'src/transactions/transferFeedUtils'
+import { TokenTransactionTypeV2 } from 'src/transactions/types'
+
+const AVATAR_SIZE = 40
+
+interface Props {
+  transfer: FeedTokenTransfer
+}
+
+export default function TransferFeedIcon({ transfer }: Props) {
```

Comment:
can you explain why we need this component in addition to the existing `src/transactions/TransferFeedIcon.tsx`?

Reply:
There seems to be separate set of transaction feed components, a set that works with TokenTransactionTypeV2 and another with TokenTransactionType. The new TransferFeedIcon works with TokenTransactionTypeV2, so following the trend of files under /transaction/feed, I made the new component.